.github/workflows/fhir-validation.yml

```diff
@@ -1,7 +1,9 @@
-name: fhir
+name: FHIR Definition Validation
```

Comment:
the github workflow will now read `FHIR Definition Validation / register` -- might be worth just deleting this entirely since noone is really using it anymore

Reply:
removing this file. should we remove FHIR from `/fern/apis/fhir` too?

Reply:
yup that makes sense

Reply:
removed.